documentation/meta/incidents/index.md

```diff
@@ -0,0 +1,160 @@
+
+As you might expect, incidents can be _destabilized_ if they reoccur, and the
+state should rollback to "Stabilization pending":
+
+```{mermaid}
+graph LR
+    A[Stabilization pending]
+    B[Stabilized]
+    C[Under investigation]
+    D[Resolved]
+
+    A --> B
+    B --> C
+    B --> A
+    C --> A
+    C --> D
+    D --> A
+```
+
+## How do we _handle_ incidents?
+
+```{warning}
+Dealing with an incident now? Go to the [Incident Response Runbook](/meta/incidents/runbooks/incident_response_runbook).
+```
+
+This section addresses the philosophy of our approach to incident management and
+explanation behind the steps in the
+[runbook](/meta/incidents/runbooks/incident_response_runbook).
+
+There are four main steps to handling production incidents:
+
+<!-- prettier-ignore -->
+| # | Name      | Goal                                                                                                                                                                                             | Status After                                         | Tag                  |
+|---|-----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------|----------------------|
+| 1 | Record    | Visibly acknowledge the incident and create the communication threads necessary for ongoing discussion and investigation. Understand how bad the situation is based on our severity definitions. | Stabilization pending OR Stabilized (as appropriate) | `#status-recorded`   |
+| 2 | Stabilize | <p>If there is a total or partial outage, identify service stabilization options and action them.                                                                                                | Stabilized                                           | `#status-stabilized` |
+| 3 | Resolve   | Identify outstanding improvements to the service that would mitigate the issue in the future and create tickets to implement.                                                                    | Under investigation                                  | `#status-resolved`   |
+| 4 | Review    | Identify process improvements. If the underlying cause is still unknown, identify further steps to resolve or continue investigation of the issue via 5-whys.                                    | Under investigation OR Resolved (as appropriate)     | `#status-reviewed`   |
+
+All maintainers are required to record incidents they encounter. **Crucially,
+maintainers who record incidents are not _inherently_ responsible for
+stabilizing them.** Their only required job is to record the incident. Every
+incident has a **Lead** who is the person responsible for providing updates on
+the incident and delegating tasks. The Lead is also responsible for hosting the
+retrospective after the incident is resolved.
```

Comment:
This qualification is repeated from above. We should qualify this once and if necessary refer or link to the specific section later, rather than re-stating it several times.